### ISSUE TITLE
Tuning the K8S resources

### ISSUE THREAD

**Issue by MoisesGSalas** [openedx/openedx-k8s-harmony#39]
# Description 

When running a somewhat large instance of Open edX in Kubernetes we need to perform several changes to the Kubernetes resources.

At first we needed to specify cpu/memory resources in the LMS/CMS/Workers pods to allow the usage of [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/). An initial [PR](https://github.com/overhangio/tutor/pull/677) was opened in Tutor but was eventually discarded due to the added complexity to Tutor Core. This eventually was implemented as an [external plugin](https://github.com/eduNEXT/tutor-contrib-pod-autoscaling).

At eduNEXT we also found ourselves making changes to the UWSGI configuration to improve the behavior of the container in the context of a kubernetes Pod. I managed to land a [PR](https://github.com/overhangio/tutor/pull/808) that allows to configure the UWSGI process.

I don't think these are going to be the only changes we are going to be making to the K8S resources, so this issues' purpose is two fold: To discuss where would be the best place to gather all that configuration and how we implement it (the harmony tutor plugin an additional plugin, some other way?), and to share and discuss those special 'tunings' that we have been applying.

**Comment by felipemontoya** [comment]
After 3 years, of using harmony successfully we now have confidence in the defaults from ~~https://pypi.org/project/tutor-contrib-pod-autoscaler~~ https://pypi.org/project/tutor-contrib-pod-autoscaling/.

Tunning is possible by altering the variables in there. 

Closing as per our WG-meeting

**Comment by uhlhosting** [comment]
@felipemontoya We looked everywhere but couldn't find this page on the package you mentioned above. Could it be the package is https://pypi.org/project/tutor-contrib-pod-autoscaling/ .

**Comment by felipemontoya** [comment]
@uhlhosting you are correct. Sorry for posting the link wrong. https://pypi.org/project/tutor-contrib-pod-autoscaling/ is the one. The tunned defauts can be found at https://github.com/eduNEXT/tutor-contrib-pod-autoscaling/blob/main/tutorpod_autoscaling/plugin.py#L35
